### ISSUE TITLE
What is database and where i see it?

### ISSUE THREAD

**Issue by Delakith** [TheAnthonyNL/SteamBot#46]
What is database and where i see it?

**Comment by IBeRamen** [comment]
You need a MySQL database to use this with CSGOWinBig

**Comment by atannerd** [comment]
To be clear - the bot does not connect to the database, it just uses the database password as a check in the CSGOWinBig code.

**Comment by IBeRamen** [comment]
Yes of course. Only the PHP files connect to the MySQL database.
